js/components/operatorSignIn/operatorSignInModal.tsx

```diff
@@ -80,10 +80,10 @@ export const OperatorSignInModal = ({
   }, [complete, close]);
 
   const employee =
-    employees.status === "ok" &&
-    badge !== null &&
-    findEmployeeByBadge(employees.result, badge.number);
-  const name = employee ? employee.first_name : `Operator ${badge?.number}`;
+    employees.status === "ok" && badge !== null ?
+      findEmployeeByBadge(employees.result, badge.number)
+    : null;
+  const name: string = employee?.first_name ?? `Operator ${badge?.number}`;
```

Comment:
I was wondering if we should DRY this? It's a little different than `lookupDisplayName` because it only uses the first name.

Reply:
Eh I think it's fine given that, as you point out, it's a bit different.